mkt/ratings/helpers.py

```diff
@@ -13,15 +12,7 @@ def stars(num, large=False):
         return _('Not yet reviewed')
     else:
         num = min(5, int(round(num)))
-        t = jingo.env.get_template('ratings/reviews_rating.html')
-        # These are getting renamed for contextual sense in the template.
-        return jinja2.Markup(t.render({'rating': num, 'detailpage': large}))
-
-
-@jingo.register.function
-def impala_reviews_link(addon):
-    t = jingo.env.get_template('ratings/reviews_link.html')
-    return jinja2.Markup(t.render({'addon': addon}))
+        return _('Rated %s out of 5 stars' % num)
```

Comment:
I think we can blame that on the schizophrenic rating/review terminology on Marketplace front end (and back end).  I'm tempted to use 'Review' as that's at what we're labelling them for consumers, for now at least.